src/lib/getTxLink.ts

```diff
@@ -33,7 +33,7 @@ type GetTxLink = GetTxBaseUrl &
   ({ txId: string; tradeId?: never } | { tradeId: string; txId?: never }) & {
     // TODO(gomes): make me required in a follow-up and have SAFE TxIds working holistically
```

Comment:
applicable or still meant as follow up?

Reply:
Nice catch! Removed comment and made it required in 9dbfbe751b9b05d230ef09a09e0e517c4cb4f218